dev/breeze/src/airflow_breeze/global_constants.py

```diff
@@ -382,57 +382,64 @@ def get_default_platform_machine() -> str:
     CURRENT_MYSQL_VERSIONS = [*MYSQL_OLD_RELEASES, *MYSQL_LTS_RELEASES]
 DEFAULT_MYSQL_VERSION = CURRENT_MYSQL_VERSIONS[0]
 
+PYTHON_3_6_TO_3_8 = ["3.6", "3.7", "3.8"]
+PYTHON_3_6_TO_3_9 = ["3.6", "3.7", "3.8", "3.9"]
+PYTHON_3_6_TO_3_10 = ["3.7", "3.8", "3.9", "3.10"]
+PYTHON_3_7_TO_3_11 = ["3.7", "3.8", "3.9", "3.10", "3.11"]
+PYTHON_3_8_TO_3_11 = ["3.8", "3.9", "3.10", "3.11"]
+PYTHON_3_8_TO_3_12 = ["3.8", "3.9", "3.10", "3.11", "3.12"]
+
 
 AIRFLOW_PYTHON_COMPATIBILITY_MATRIX = {
```

Comment:
is this being updated automatically?
if not we need to add a step in core release guide to update this

Reply:
No. It's not easy to derive automatically, and it's almost no effort to keep it updated. so we can leave it as it is.

Reply:
I didn't mean automation, manual step for release manager to follow when he raises the post release PR to change all defaults